### PR TITLE
Release notes for 17.2.0

### PR DESCRIPTION
**Pull request by iamsumanth**

### CODE REVIEW

File: source/partials/release_notes/_release-17-2-0.md.erb [hunk 1]
```diff
@@ -0,0 +1,119 @@
+#### Authorization Plugins (beta)
+
+Currently GoCD provides multiple ways of Authentication like File based Authentication, LDAP/AD Authentication and Plugin Based Authentication. Apart from Authentication, GoCD provides ability to restrict certain operations to specific Users and Roles and manage them. Currently, roles can be managed only through GoCD and it does not provide an ability to use ACL’s defined in systems used for authentication. With this limitation, administrators need to configure ACL's in multiples places.
+
+With the introduction of Authorisation Plugin(Beta)  GoCD is moving in a direction where Authentication/Authorisation is moved out of GoCD into plugins. With this, the Authorisation Plugins would be responsible to provide Identity Service. The plugins have the flexibility to use any identity service providers like LDAP, Github etc.
```
Comment (GaneshSPatil) [inline source/partials/release_notes/_release-17-2-0.md.erb]: can you indent the spaces between *Authorisation Plugin(Beta)  GoCD* and change `Authorisation` to `Authorization` ?

File: source/partials/release_notes/_release-17-2-0.md.erb [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+
+[LDAP Authentication plugin](https://github.com/gocd/gocd-ldap-authentication-plugin) is in place built on this Authorization endpoint. Please checkout this feature and provide us feedback to enhance this endpoint.
+
+#### Feature Enhancements
```
Comment (GaneshSPatil) [inline source/partials/release_notes/_release-17-2-0.md.erb]: can we add few more things in feature enhancements ?(if there are any or else its fine)

File: source/partials/release_notes/_release-17-2-0.md.erb [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+<figure>
+  <%= image_tag 'releases/17.2.0/template-permissions.png', alt: 'Template Permissions' %>
+</figure>
+
+#### Bug Fixes
+
+* <%= link_to_commit '7ddf84eaaf813546aa1f8eb684ef9cf3187f6617', 'User search functionality - Search through plugins implementing AuthenticationExtension.', 3107 %>
+* <%= link_to_commit 'a837fc05cca60a9bbb8dcd7d3a3c7b1228a51358', 'Sort resources and environments alphabetically on agents page selector dropdown.', 3134 %>
+* Multiple UI fixes on the old and new themes.
```
Comment (GaneshSPatil) [inline source/partials/release_notes/_release-17-2-0.md.erb]: can  we add github issue(s)/PR(s) link?

File: source/partials/release_notes/_release-17-2-0.md.erb [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  * [Maven Task Plugin](https://github.com/ruckc/gocd-maven-plugin)
+  * [Publish to Artifactory](https://github.com/tusharm/go-artifactory-plugin)
+
+
+
+
+
+* <%= link_to_commit 'a6f1fac8231a5ee90aedc0395d6c31cd3a451021', 'Removed Pipeline Config API V2', 3174 %>
+
+  As of release 16.12, version 2 of the <%= link_to_api 'pipeline-config', 'Pipeline Config API' %> is deprecated. Users are encouraged to use Version 3 of the API which is backward compatible.
```
Comment (GaneshSPatil) [inline source/partials/release_notes/_release-17-2-0.md.erb]: can we say -- As of release 16.12, version 2 of the Templates API **was** deprecated. Users are encouraged to use Version 3 of the API which is backward compatible **with Version 2**.

File: source/partials/release_notes/_release-17-2-0.md.erb [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+* <%= link_to_commit '5a208ba9fa537157fd58e2417a35333b927d0edf', 'Removed Plugin Info API V1', 3143 %>
+
+  As of release 16.12, version 1 of the <%= link_to_api 'get-all-plugin-info', 'Plugin Info API' %> is deprecated. Users are encouraged to use Version 2 of the API.
```
Comment (GaneshSPatil) [inline source/partials/release_notes/_release-17-2-0.md.erb]: same as above

File: source/partials/release_notes/_release-17-2-0.md.erb [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+* Stopped support for git versions older than 1.9.
+
+  As of release 16.12, git versions older than 1.9 are deprecated. Users are recommended to upgrade git on the GoCD server and agents (if git is used as a material). We are aware that the linux distributions have an older version of git installed by default. Please refer to [this](https://www.softwarecollections.org/en/scls/rhscl/git19/) and/or [this](http://askubuntu.com/questions/571549/git-1-7-9-5-upgrade-to-current-release-of-git-2-x-on-ubuntu-12-04/571633#571633) to upgrade git.
```
Comment (GaneshSPatil) [inline source/partials/release_notes/_release-17-2-0.md.erb]: As of release 16.12, git versions older than 1.9 **were** deprecated.

File: source/partials/release_notes/_release-17-2-0.md.erb (hunk 4 above)
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-17-2-0.md.erb]: If version 2 of the api is removed, then version 3 isn't backward compatible with any version. Just say, `Version 3 of the api is available going forward`. Users need to use it as there is no alternative.

File: source/partials/release_notes/_release-17-2-0.md.erb [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+With the introduction of Authorisation Plugin(Beta)  GoCD is moving in a direction where Authentication/Authorisation is moved out of GoCD into plugins. With this, the Authorisation Plugins would be responsible to provide Identity Service. The plugins have the flexibility to use any identity service providers like LDAP, Github etc.
+
+
+[LDAP Authentication plugin](https://github.com/gocd/gocd-ldap-authentication-plugin) is in place built on this Authorization endpoint. Please checkout this feature and provide us feedback to enhance this endpoint.
+
+#### Feature Enhancements
+
+* <%= link_to_commit 'c08843512b494ef1c483ebb699c615170dd13cf0', "Template Authorization UI and API enhancements" %>
+
+* No restriction on template view config. All pipeline group admins by default provided view permission to templates. Super admin can restrict this access.
```
Comment (varshavaradarajan) [inline source/partials/release_notes/_release-17-2-0.md.erb]: No restriction on template view config sounds a bit vague to me. Besides, it was implicitly already there for pipeline group admins. Can we just say `All pipeline group admins by default provided view permission to templates. Super admin can now restrict this access.`

File: source/partials/release_notes/_release-17-2-0.md.erb (hunk 3 above)
Comment (iamsumanth) [inline source/partials/release_notes/_release-17-2-0.md.erb]: There are multiple UI fixes, couldn't find any PR which is major in those. That's why didn't add any PR.

**Comment by jyotisingh** [comment]
@GaneshSPatil - @iamsumanth is working on some feedback, so I reverted the merge.

**Pull request merged**